src/views/ParticipantEntity/utils/getDiagnosisColumns.tsx

```diff
@@ -9,51 +11,72 @@ import { TABLE_EMPTY_PLACE_HOLDER } from 'common/constants';
 import AgeCell from '../AgeCell';
 import MondoParticipantCount from '../DiagnosisTable/MondoParticipantCount';
 
-const getDiagnosisColumns = (): ProColumnType[] => [
+const getDiagnosisDefaultColumns = (): ProColumnType[] => [
   {
     key: 'mondo_id_diagnosis',
-    dataIndex: 'mondo_id_diagnosis',
     title: intl.get('entities.participant.mondo_diagnosis'),
-    render: (mondo_id_diagnosis: string) => {
-      if (!mondo_id_diagnosis) {
-        return TABLE_EMPTY_PLACE_HOLDER;
-      }
-
-      const mondoInfo = extractMondoTitleAndCode(mondo_id_diagnosis);
-
+    render: (diagnosis: IParticipantDiagnosis) => {
+      const mondoNames = diagnosis.mondo_id_diagnosis;
```

Comment:
```suggestion
      const mondoNames = diagnosis?.mondo_id_diagnosis;
```
dis moi si tu trouves pas ca pertinent mais ca eviterait quelques erreurs non ?
idem dans tout le fichier